config.py

```diff
@@ -9,6 +10,9 @@ class BaseConfig:
     DOMAIN_URL = os.getenv("DOMAIN_URL")
     DOMAIN_URL_PROTOCOL = os.getenv("DOMAIN_URL_PROTOCOL")
 
+    SECRET_KEY = os.getenv("SECRET_KEY")
+    SESSION_COOKIE_SECURE = bool(strtobool(os.getenv('SESSION_COOKIE_SECURE', 'True')))
```

Comment:
```suggestion
```
Dupicates

Reply:
I think they're both necessary, the base config and development config need different defaults.q